docs/pages/admin-guides/access-controls/idps/saml-guide.mdx

```diff
@@ -34,15 +34,10 @@ of the service provider.
 
 Below we'll show both of the configuration options.
 
-First, in the Web UI, under **Access Management**, click **Enroll New Resource** menu.
-In the search box, enter "saml", which will show the SAML application
-integration tile. Click the tile.
+First, in the Web UI, under **Zero Trust Access**, click **Auth Connectors** in the menu.
+Choose the appropriate SAML application integration tile. Click the tile and follow the resulting steps.
 
-![Enroll SAML app](../../../../img/access-controls/saml-idp/enroll-saml-app.png)
-
-The first configuration step, **Configure Service Provider with Teleport's Identity Provider Metadata**
-shows Teleport SAML IdP metadata values. For this guide, you can move to next step by clicking **Next** button
-which takes to **Add Service Provider To Teleport** step.
+![Enroll SAML app](../../../../img/access-controls/saml-idp/saml-idp.png)
```

Comment:
Would it be possible to make this image sharper and use a horizontal layout? It seems a little blurry and takes up a lot of room on the screen.